### PR TITLE
Modify some text to be more pretty

### PR DESCRIPTION
**Pull request by penut85420**
* Modify README.md
  * Erase meaningless tags despite it is useful in HackMD.
  * Maintain a table of content to be compatible with GitHub.
  * Fix some capitalization issues.
  * Remove redundant spaces.
* Modify myconfig.java
  * Change simplified Chinese into traditional Chinese.
* Modify DiscordService.java
  * Maintain consistent bracket style.
  * Remove redundant spaces.
* Modify index.html
  * Change lang of html into zh-Hant-TW.
  * Fix some capitalization issues.
  * Maintain consistent bracket style.

### CODE REVIEW

**Comment by DF-wu** [comment]
@penut85420  Thanks for your contribution.
- Here are some issues, plz resolve it then I can approve the PR.
  1. There is a conflict in src/main/resources/static/index.html, plz synchronize with the latest commit.
  2. About the READMD.md file. As a result of I use HackMD to manage it, I am not sure if there were better solution to keep the view best between the two different platforms .

**Comment by DF-wu** [comment]
話說 要怎麼嵌入commit內容和request change阿
跟https://github.com/penut85420/TranslationImprover/pull/3 你要求我的一樣

File: src/main/resources/static/index.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <div class="topPanel">
         <h1>匿名回覆機器人</h1>
         [<b><a href="https://hackmd.io/@dfder/ByADyqspI" target="_blank">使用說明</a></b>]
-        [<b><a href="https://github.com/DF-wu/HideReplier" target="_blank">Github</a></b>]
+        [<b><a href="https://github.com/DF-wu/HideReplier" target="_blank">GitHub</a></b>]
         <b>←給個星星牡蠣一下好嗎？</b>
```
Comment (DF-wu) [inline src/main/resources/static/index.html]: - conflict
- plz merge the latest commit

File: README.md [hunk 2]
```diff
@@ -1,33 +1,39 @@
----
-tags: HideReplier
----
-
```
Comment (DF-wu) [inline README.md:4]: - for HackMD
- any better solution?

**Review by DF-wu — CHANGES_REQUESTED** [review]

File: src/main/resources/static/index.html (hunk 1 above)
Comment (penut85420) [reply src/main/resources/static/index.html]: Solved by 10506f0

File: README.md (hunk 2 above)
Comment (penut85420) [reply README.md:4]: You should push to a new branch from HackMD, and modify the markdown file to be compatible with GitHub, then merge into master. [Here](https://github.com/penut85420/HelloHackMD) is a demonstration repo.

Comment (DF-wu) [reply README.md:4]: Got it! thx

**Review by DF-wu — APPROVED** [review]
thx @penut85420 's contribution.

**Pull request merged**